tests/js/spec/views/performance/vitalDetail/index.spec.tsx

```diff
@@ -31,7 +31,7 @@ const {
   router: {
     location: {
       query: {
-        project: 1,
+        project: '1',
```

Comment:
had to change these to strings because when we extract values from query parameters in `actionCreators/pageFilters.tsx` the type we can receive from the project query field is constrained to `string | string[] | null | undefined`. 

Not sure if I should make the code resilient to receiving `number` here since it seems like that isn't possible as a query parameter.